books/free-programming-books-langs.md

```diff
@@ -1457,6 +1458,16 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Modelica by Example](http://book.xogeny.com)
 
 
+### MongoDB
+
+* [Introduction to MongoDB](https://www.tutorialspoint.com/mongodb) - Tutorials Point
+* [MongoDB Koans](https://github.com/chicagoruby/MongoDB_Koans)
+* [MongoDB Notes for Professionals](https://goalkicker.com/MongoDBBook/) - Compiled from StackOverflow Documentation (PDF)
+* [MongoDB Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/mongodb) (PDF, Kindle) (email address *requested*, not required)
+* [The Definitive Guide to MongoDB](https://pepa.holla.cz/wp-content/uploads/2016/11/The-Definitive-Guide-to-MongoDB.pdf) - Eelco Plugge, Peter Membrey and Tim Hawkins (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines. Remove it, or explain how we know it's an authorized distribution.